terminology_bot/bot.py

```diff
@@ -0,0 +1,386 @@
+import logging
+import hashlib
+from telegram import (ReplyKeyboardMarkup, ReplyKeyboardRemove)
+from telegram.ext import (Updater, CommandHandler, MessageHandler, Filters, RegexHandler,
+                          ConversationHandler)
+
+from config import get_config
+from term_collection import TermCollection
+from database import POSEnum
+
+
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+                            level=logging.INFO)
+
+logger = logging.getLogger(__name__)
+params = get_config(section='bot')
+
+
+class Bot:
+    START_MENU, CHOOSE_TERM, NEW_TERM, CHOOSE_OPTION, \
+    POS, DESCRIPTION, SYNONYMS, SIMILARS, IMAGE, AUDIO, VIDEO = range(11)
+
+    reply_start_keyboard = ReplyKeyboardMarkup([['Add new term'], ['Get list of terms']], resize_keyboard=True)
+    reply_menu_keyboard = ReplyKeyboardMarkup(
+        [['POS-tag', 'Description'],
+         ['Synonyms', 'Similar words'],
+         ['Image', 'Audio', 'Video']],
+        resize_keyboard=True
+    )
+
+    def __init__(self):
+        self.updater = Updater(token=params['token'])
+        self.dispatcher = self.updater.dispatcher
+        self.term_collection = TermCollection()
+        self.cur_term = {}
+
+    def start(self, bot, update):
+        """
+        Sends the greeting message with the start menu: 'Add new term' and 'Get list of terms' options
+        :return: the state START_MENU
+        """
+        update.message.reply_text('Hello! I am Terminology Bot. Send /cancel to stop talking to me.',
```

Comment:
I'm sure that good practise is to use i18n to have (in future) abitlity to translate our UI (https://docs.python.org/3/library/i18n.html, gettext)

Please create an issue to backlog to cover all UI-related strings by i18n calls.

Reply:
The issue is created.